src/posts/integrating-lighthouse-ci/integrate-lighthouse-ci.md

```diff
@@ -66,8 +66,6 @@ before_install:
 script:  
  - yarn run build # command in package.json to build your site  
 after_success:  
- - npx http-server -s -p 8080 ./public &
```

Comment:
autorun will do these steps for you already

Reply:
if you specify URLs you'll need to explicitly set your `staticDistDir`, sorry forget to add that edit to the lighthouserc file :)

Reply:
Thanks, worked like a charm for me now. But, I think we should specifically mention this (to add `staticDistDir` in case custom URLs are added) in the documentation of LHCI. 
Also, I didn't have understanding that LHCI will itself extract the relative part from the URL and check on it when `urls` are mentioned as it's running on different port than `80`.